clvm/serialize.py

```diff
@@ -1,15 +1,17 @@
 # decoding:
 # read a byte
-# if it's 0xfe, it's nil (which might be same as 0)
+# if it's 0x80, it's nil (which might be same as 0)
 # if it's 0xff, it's a cons box. Read two items, build cons
 # otherwise, number of leading set bits is length in bytes to read size
-# 0-0x7f are literal one byte values
-# leading bits is the count of bytes to read of size
-# 0x80-0xbf is a size of one byte (perform logical and of first byte with 0x3f to get size)
-# 0xc0-0xdf is a size of two bytes (perform logical and of first byte with 0x1f)
-# 0xe0-0xef is 3 bytes ((perform logical and of first byte with 0xf))
-# 0xf0-0xf7 is 4 bytes ((perform logical and of first byte with 0x7))
-# 0xf7-0xfb is 5 bytes ((perform logical and of first byte with 0x3))
+# For example, if bit fields of the reading first byte are:
```

Comment:
```suggestion
# For example, if the bit fields of reading first byte are:
```

I apologize, I made a typo in my previous suggestion. And I am a native English speaker. 😀